docs/source/release/v6.5.0/index.rst

```diff
@@ -16,7 +16,25 @@ Mantid 6.5.0 Release Notes
 
 We are proud to announce version 6.5.0 of Mantid.
 
-**TODO: Add paragraph summarizing big changes**
+This release includes a wide array of updates, bugfixes, and new features, which build on the functionality of Mantid.
+Users are reminded that since the previous release the :ref:`packaging method for Linux has changed <v6.4.0>`.
```

Comment:
I like that we have this reminder. Can we make it more explicit though? Something like:

*Users are reminded that this release has now dropped support for `.rpm` & `.deb` files on Linux in favour of Conda installations or a standalone tarball. Please see the :ref:`6.4.0 release notes <v6.4.0>` for more information.*